### ISSUE TITLE
[Feature Request] Custom index gestures or capture other knuckles capacitors

### ISSUE THREAD

**Issue by Livb-VRC** [benaclejames/VRCThumbParams#2]
Would be really nice if you could capture custom hand gestures like for example pinch/okhand(thumb resting on thumbpad + thumb on trigger + no fingers resting on the controller) or any other easy ones that come to mind. Rock n roll and victory are hard to do reliably in vrchat so this would come in super handy for a lot of people. That or add the option to capture other knuckles capacitors for example resting your index finger on the trigger, squeezing hand etc

**Comment by InsaneGrox** [comment]
agreed, I prefer the look of gestures over finger tracking and don't want to be limited to just the 7 gestures for my personal preference...
This seems like this should be it's own mod in it's own right however...
